### PR TITLE
Fix console selector for ESS client docs

### PR DESCRIPTION
**Pull request by nrichers**
This PR fixes the language selector dropdown for the Elasticsearch Service docs. Previously, no selector was shown at all. With a small fix to conf.yaml, the selector now appears as expected:

image

Before this is ready for review: 

* With the selector enabled, none of the language examples can be found. Need to investigate further to see what's amiss. ~~Possibly related to us needing to map to the `master` branch in the clients-team repo?~~ EDIT: After investigating further, I think this might be because of how the examples in the elastic/cloud repo are written, which will require a separate PR to fix. 

  image

* Added the `master` branch to the ESS doc build for testing; need to remove before merging

Also added an alias for `docbldess`. Unrelated but helpful, since `docbldec` is dates back to the pre-renaming days of our SaaS.

### CODE REVIEW

**Comment by philkra** [comment]
good catch with changing `csharp` back to ` console`, I forgot that one! thanks

**Comment by nrichers** [comment]
@philkra can you approve this PR? I think the remaining fix will need to go into the elastic/cloud repo.

File: conf.yaml [hunk 1]
```diff
@@ -626,33 +626,33 @@ contents:
                 repo:   cloud
                 path:   docs/shared
               -
```
Comment (philkra) [inline conf.yaml]: we're missing .NET, need to add the following block:
```
alternatives: { source_lang: console, alternative_lang: csharp }
repo:   clients-team
path:   examples/elastic-cloud/csharp
map_branches: *mapCloudSaasToClientsTeam
```

Comment (nrichers) [reply conf.yaml]: Fixed, thank you for pointing this out!

**Comment by nik9000** [comment]
run elasticsearch-ci/docs

File: conf.yaml [hunk 2]
```diff
@@ -751,30 +761,35 @@ contents:
...
-                alternatives: { source_lang: csharp, alternative_lang: python }
+                alternatives: { source_lang: console, alternative_lang: python }
                 repo:   clients-team
                 path:   examples/elastic-cloud/python
                 map_branches: *mapCloudEceToClientsTeam
+              -
+                alternatives: { source_lang: console, alternative_lang: csharp }
+                repo:   clients-team
+                path:   examples/elastic-cloud/csharp
+                map_branches: *mapCloudSaasToClientsTeam
```
Comment (nik9000) [inline conf.yaml]: s/mapCloudSaasToClientsTeam/mapCloudEceToClientsTeam/

Comment (nrichers) [reply conf.yaml]: Good catch! In my pre-meeting hurry to address Philip's comment, I messed this up. Fixed now.

**Comment by nik9000** [comment]
Looks like it is working now!

**Comment by nrichers** [comment]
Tested ESS, Heroku and ECE docs to make sure everything works as expected. Removed `master` from conf.yaml. This PR is ready to merge.

**Pull request merged**